.github/workflows/pr.yml

```diff
@@ -0,0 +1,17 @@
+name: Node.js CI
+
+on: [push]
```

Comment:
What do you think about changing this to on: pull_request?

On push seems to aggressive and will over utilize resources.